packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -134,6 +134,17 @@ export class ConnectionPropsForSessCfg {
             }
         }
 
+        // Set default values on propsToPromptFor
+        if(connOpts.propsToPromptFor?.length > 0)
+        {
+            connOpts.propsToPromptFor.forEach(obj => {
```

Comment:
My apologies for the confusion.
During the Zowe CLI standup (Aug 5th) I mentioned that this could be ommitted for coverage purposes.

I was wrong. For some reason I was under the impression that the sonar cloud reports where complaining about the SshBaseHandler statements not being covered.

I think it would be good to add a couple of tests for this 🙏🏽